programmers-coding-test/jaeseong/greedy/1이-될때까지.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+const rl = require("readline-sync");
+let [ n,k ] = rl.question("n m input please : \n>>").split(" ");
+console.log(`\n입력값 :${n},${k}`);
+
+function solution() {
+    console.log(n,k);
+    let count = 0;
+    while (n !== 1) {
+        if ( n % k !== 0) {
+            n -= 1;
+            count++;
+        } else if (n % k === 0){
+            n /= k; 
+            count++;
+        }
```

Comment:
삼항 연산으로 간추릴 수 있습니다
또한 조건이 두개 밖에 없으므로 else if 보단 else가 이해하기 좋을 것 같고, count는 어떤 조건이든 증감시켜줘야하므로 조건 밖으로 빼주는 것이 더 좋습니다. 중복은 제거하는 것이 좋으니까요 ㅎㅎ
```suggestion
       n = n % k ? n - 1 : n / k;  // N이 K로 나눠지지 않으면 N에서 1을 빼고, 나눠지면 N / K를 수행하여 N에 대입한다.
       cnt++;
```